source/MaterialXContrib/MaterialXNode/MaterialXData.cpp

```diff
@@ -34,60 +39,62 @@ MaterialXData::~MaterialXData()
 
 const std::string& MaterialXData::getFragmentName() const
 {
-    return _xmlFragmentWrapper.getFragmentName();
-}
-
-void MaterialXData::getXML(std::ostream& stream) const
-{
-    _xmlFragmentWrapper.getXML(stream);
-}
-
-const MaterialX::StringVec&  MaterialXData::getGlobalsList() const
-{
-    return _xmlFragmentWrapper.getGlobalsList();
+    return _fragmentName;
 }
 
-const MaterialX::StringMap& MaterialXData::getPathInputMap() const
+const std::string& MaterialXData::getFragmentWrapper() const
 {
-    return _xmlFragmentWrapper.getPathInputMap();
+    return _fragmentWrapper;
 }
 
-const MaterialX::StringMap& MaterialXData::getPathOutputMap() const
+const mx::StringMap& MaterialXData::getPathInputMap() const
 {
-    return _xmlFragmentWrapper.getPathOutputMap();
+    return _pathInputMap;
 }
 
 void MaterialXData::createDocument(const std::string& materialXDocumentPath)
 {
     // Create document
-    _document = MaterialX::createDocument();
+    _document = mx::createDocument();
```

Comment:
@ppenenko as code is touched will move to using mx as a namespace shortcut for code consistency with other mtlx code.